source/connection_manager.c

```diff
@@ -678,7 +686,8 @@ static void prvMQTTAgentTask( void * pvParameters )
             configASSERT( xNetworkResult == pdPASS );
             pMqttContext->connectStatus = MQTTNotConnected;
             /* MQTT Connect with a persistent session. */
-            xMQTTStatus = prvMQTTConnect( false );
+            xConnectStatus = prvMQTTConnect( false );
+            configASSERT( xConnectStatus == MQTTSuccess );
```

Comment:
Why are there asserts for these network calls?

Reply:
This is part of the demo, and we've been in the practice of using config asserts for demos